core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -136,6 +136,15 @@ private[spark] class Executor(
   // for fetching remote cached RDD blocks, so need to make sure it uses the right classloader too.
   env.serializerManager.setDefaultClassLoader(replClassLoader)
 
+  private val pluginList = conf.get(EXECUTOR_PLUGINS)
+  if (pluginList != Nil) logDebug(s"Loading the following plugins: ${pluginList.mkString(", ")}")
```

Comment:
`pluginList.nonEmpty`

Also we generally use the full syntax with braces, unless it's a one line `if..else`.